blocks/columns/columns.css

```diff
@@ -19,15 +15,93 @@
   display: block;
 }
 
+/*
+ * Column Manufacturing Steps
+*/
+
+.section.columns-container h2 {
```

Comment:
this is outside the columns block scope. Do we feel that this font size will be used for `h2` everywhere? if so we can move it to `styles` and not provide section/column context (i.e. make it global/default)

Reply:
It made sense going in, but that rule is also on every h2 in the block on the next rule.  I think that's good enough.  I was just standardizing `h2` sizes based on the current site so that we could change the `font-size` for all `h2` headers at the same time if needed.